migration/config/spire-controller-manager-config.yaml

```diff
@@ -0,0 +1,25 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: spire-controller-manager-config
+  namespace: spire
+data:
+  spire-controller-manager-config.yaml: |
+    apiVersion: spire.spiffe.io/v1alpha1
+    kind: ControllerManagerConfig
+    metrics:
+      bindAddress: 127.0.0.1:8082
+    healthProbe:
+      bindAddress: 127.0.0.1:8083
+    leaderElection:
+      leaderElect: true
+      resourceName: 98c9c988.spiffe.io
```

Comment:
What is this name referring to?

Reply:
Good question, its the name of the `ConfigMap` thats created to do leader election. Its defined here: https://pkg.go.dev/k8s.io/component-base/config/v1alpha1#LeaderElectionConfiguration

Reply:
Will it always have this name?

Reply:
You can use a different name if you like.

Reply:
Oh I see, wasn't actually sure who generated that ConfigMap. Good to know that we control the name.

Reply:
I did following in my Helm chart.

```yaml
resourceName: {{ .Release.Name | sha256sum | trunc 8 }}.spiffe.io
```